### PR TITLE
ENCD-4279 Biosample preservation

### PR DESCRIPTION
**Pull request by weiweizhong2018**

### CODE REVIEW

**Comment by jahilton** [comment]
I think this should get a new test or be incorporated into one of the existing biosample_summary tests.

File: src/encoded/types/experiment.py [hunk 1]
```diff
@@ -223,6 +223,7 @@ def biosample_summary(self,
                                 biosampleObject.get('post_treatment_time'),
                                 biosampleObject.get('post_treatment_time_units'),
                                 treatment_objects_list,
+                                None,
```
Comment (jahilton) [inline src/encoded/types/experiment.py]: Can you explain the purpose of the addition of this line?

Comment (gabdank) [reply src/encoded/types/experiment.py]: this line is to eliminate the specification of preservation method in the experiment biosample summary. It was added per my request, trying to keep the experiment level summaries terse.

Comment (weiweizhong2018) [reply src/encoded/types/experiment.py]: The generate_summary_dictionary method in biosample is changed to have preservation_method as an input parameter. Here in the experiment class, this method is called to display biosample summary in the experiment summary. Therefore, we need to supply a value for the preservation_method parameter here. Idan said that he does not want preservation_method to be displayed in biosample.summary of the experiment, so it is set to 0. I am going to change it to two lines of code: preservation_method=None and then pass preservation_method as a parameter.

Comment (weiweizhong2018) [reply src/encoded/types/experiment.py]: I added preservation_method in test_biosample_summary.

Comment (jahilton) [reply src/encoded/types/experiment.py]: Thank you for clarifying - makes sense (especially with the code update)

**Pull request merged**